wandb/sdk/launch/builder/noop.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, Optional
+
+from wandb.errors import LaunchError
+from wandb.sdk.launch.builder.abstract import AbstractBuilder
+
+from .._project_spec import EntryPoint, LaunchProject
+
+
+class NoOpBuilder(AbstractBuilder):
+
+    type: str
+
+    def __init__(self, builder_config: Dict[str, Any]) -> None:
+        self.builder_config = builder_config
+
+    def build_image(
+        self,
+        launch_project: LaunchProject,
```

Comment:
Do we use pylint? I would have expected (ignorable) warnings about unused arguments here.

Reply:
We don't use pylint in our linting check. For a comprehensive list of our linting, checkout: https://github.com/wandb/wandb/blob/master/.circleci/config.yml#L1089